src/packages/dto/validateOptionsSchema.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+
+export const validateOptionsSchema = z.object({
+  name: z.string().min(1, { message: "" }),
```

Comment:
We can set rules for things that are too broad like this. Think of the FE usability and database storage. What should be the limits here?

Reply:
I deleted this file as it has no use in the code

change here [612983b](https://github.com/podcodar/voting-system/pull/67/commits/612983bf43d4052ff6fd484a1995dba2bd92b7c1)